Java/E-montazysta/src/main/java/com/emontazysta/model/dto/ToolDto.java

```diff
@@ -15,8 +16,10 @@
 public class ToolDto {
 
     private Long id;
+    @NotBlank(message = "Name cannot be empty")
     private String name;
     private LocalDate createdAt;
+    @NotBlank(message = "Code cannot be empty")
```

Comment:
```suggestion
```
Code is created on backend while creating new element.
https://github.com/s20146/PRO/blob/23a8b9a0ff4f1cd696ca79848103ed664bd4da22/Java/E-montazysta/src/main/java/com/emontazysta/service/impl/ToolServiceImpl.java#L57